client-linuxapp/src/main.rs

```diff
@@ -498,6 +501,16 @@ async fn main() -> Result<()> {
         log::info!("Device credential deactivated, skipping Device Onboarding");
         return Ok(());
     }
+    if dc.protocol_version() != ProtocolVersion::Version1_1 {
+        log::info!(
```

Comment:
do we need the log here? doesn't bailing suffice as it's going to be logged anyway?

Reply:
Yep.